src/CameraDevice.ts

```diff
@@ -160,6 +176,10 @@ export interface CameraDeviceFormat {
    * Specifies whether this format supports HDR mode for photo capture
    */
   supportsPhotoHDR: boolean;
+  /**
+   * Specifies whether this format supports depth data capture
+   */
+  supportedDepthDataFormats: DepthDataFormat[];
```

Comment:
Please make the Android func return `[]` (aka no depth data formats are supported). Change is required in `CameraViewModule.kt`